src/bindings/jna/README.md

```diff
@@ -1,43 +1,121 @@
-A full java binding using JNA.
+A full java binding using JNA. This binding requires elektra to be installed on 
+the system to work.
 
 ## Usage ##
 
+To use the bindings in a java project, we have to include the jar file 
+libelektra-0.8.19.jar in the project. This jar is created upon build, if you 
```

Comment:
Please keep this name generic with globbing chars (`*`) so that we do not have to update the text every version.

(Several more occurrences below)